cmd/aws/cli/main.go

```diff
@@ -14,6 +14,7 @@ package main
 import (
 	"github.com/snowplow/snowbridge/cmd/cli"
 	"github.com/snowplow/snowbridge/config"
+	"github.com/snowplow/snowbridge/pkg/batchtransform/batchtransformconfig"
```

Comment:
We have already `github.com/snowplow/snowbridge/pkg/transform` so what do you think about making batch part of it, like: `"github.com/snowplow/snowbridge/pkg/transform/batch/....`?

Reply:
Hmm good point - batch transformations are a different class of thing to my mind - hence why I did it this way to begine with. 

But from a project organisation point of view it might make more sense to organise things so that we have `transform/batch` and `transform/{the existing ones}`. Major downside is that we now need to come up with a name for that :D lmk if you have ideas

Reply:
`single`....? `batchbutwithsize1`...? I give up, let's leave it :p

Reply:
`notbatch` it is